packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -96,116 +106,159 @@ const ChartConfigTabs: FC = () => {
         selectedAxisInSeries.length === 1;
     const selectedAxisIndex = selectedAxisInSeries[0] || 0;
 
+    const yAxisLabel = getAxisName({
+        isAxisTheSameForAllSeries,
+        selectedAxisIndex,
+        axisReference: 'yRef',
+        axisIndex: 0,
+        series: dirtyEchartsConfig?.series,
+        items,
+    });
+
     return (
         <Wrapper>
             <Tabs
                 onChange={setTab}
                 selectedTabId={tab}
                 renderActiveTabPanelOnly
             >
-                <Tab
-                    id="layout"
-                    title="Layout"
-                    panel={<FieldLayoutOptions items={items} />}
-                />
-                <Tab
-                    id="series"
-                    title="Series"
-                    panel={
-                        pivotDimension ? (
-                            <GroupedSeriesConfiguration
-                                items={items}
-                                layout={dirtyLayout}
-                                series={dirtyEchartsConfig?.series}
-                                getSeriesColor={getSeriesColor}
-                                updateSingleSeries={updateSingleSeries}
-                                updateAllGroupedSeries={updateAllGroupedSeries}
-                            />
-                        ) : (
-                            <BasicSeriesConfiguration
-                                items={items}
-                                layout={dirtyLayout}
-                                series={dirtyEchartsConfig?.series}
-                                getSeriesColor={getSeriesColor}
-                                updateSingleSeries={updateSingleSeries}
-                            />
-                        )
-                    }
-                />
-                <Tab
-                    id="axes"
-                    title="Axes"
-                    panel={
-                        <>
-                            <InputWrapper
-                                label={`${
-                                    dirtyLayout?.flipAxes ? 'Y' : 'X'
-                                }-axis label`}
-                            >
-                                <InputGroup
-                                    placeholder="Enter axis label"
-                                    defaultValue={
-                                        dirtyEchartsConfig?.xAxis?.[0]?.name ||
-                                        (xAxisField && getItemLabel(xAxisField))
-                                    }
-                                    onBlur={(e) =>
-                                        setXAxisName(e.currentTarget.value)
+                {!isBigNumber && (
+                    <Tab
+                        id="layout"
+                        title="Layout"
+                        panel={<FieldLayoutOptions items={items} />}
+                    />
+                )}
+                {!isBigNumber && (
```

Comment:
I can't read it properly on github, but on line 148 there is a `:` does this mean that you are trying to use `?` instead of `&&` ?